db/migrate/20160531152256_add_payment_mode_to_conferences.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentModeToConferences < ActiveRecord::Migration
+  def change
+    add_column :conferences, :payment_mode, :string, default: 'Offline'
```

Comment:
I am not using `payment_mode` to ever be `NULL`, so I've added `null: false`  on `payment_method` as well.